### ISSUE TITLE
Some tail sprites that aren't tails clip over clothing when they shouldn't.

### ISSUE THREAD

**Issue by ShadzFen** [Eclipse-Station/NEV-Northern-Light#277]
In the species customisation menu, the tail style is not always a tail. A character that may choose to have goat legs will be cursed to eternal public indecency! This is probably cus the tail layer is forced above clothes, as tails should be, and doesn't take into account the exceptional sprites that aren't tails but are classified as such. 
![image](https://user-images.githubusercontent.com/45645502/86516935-214d4180-be25-11ea-8358-4b1f5bd77f9b.png)
How it should work is clothing such as skirts should show above the legs, but keep the goats legs sprite underneath, and clothing such as long pants should cover the whole legs down to the hooves. Pretty sure goats legs is the only sprite that has this issue, from what I can see in the list.

**Comment by EvilJackCarver** [comment]
*Technically*, this is intended behaviour... on Virgo code. One of the verbs players on Virgo code have is a button which switches your tail layer to render over or under your clothing (TAIL_LAYER and TAIL_LAYER_ALT). However, we don't have that yet.

I'm gonna be leaving this open since it's something we need to look into at some point, but it's not high-priority for now.

**Comment by ShadzFen** [comment]
Yeah I think Nestor told me he was gonna be looking into it himself. Either way it's not really high priority. I skimmed through most of the tails and I think the goat legs might actually be the only one with this issue.
